File: ruby/find.rb
numbers = [10,20,30,40,50,10]
puts "Enter the number to search"
search_number = gets.to_i

result = numbers.find { |n|n == search_number }

if result.nil?

	puts "#{search_number} not found"
else
	puts "#{search_number} is found"
end

names = ["sharath", "kumar", "harish"]
puts "enter the name to be searched"
search_name =gets.chomp

result = names.find {|name| name==search_name}

if result.nil?
	puts "#{search_name} not found"
else
	puts "#{search_name} is found"
end

File: js/Day3 Array/find.js
// find
numbers = [10,20,30,40,50];
var result = numbers.find(function(n){
	return n > 15
});
console.log(result);


var result = numbers.find(function(n){
	return n > 55
});

console.log(result);

// find_all ==  filter

/*
numbers.find_all{|n| n>=20}

*/

var results = numbers.filter(function(n){
	return n>=20;

});
console.log(results);